chall02/isoria-g.py

```diff
@@ -0,0 +1,12 @@
+import sys
+letter=("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
+morse=(".-","-...","-.-.","-..",".","..-.","--.","....","..",".---","-.-",".-..","--","-.","---",".--.","--.-",".-.","...","-","..-","...-",".--","-..-","-.--","--..")
+if len(sys.argv) == 2 and (all(c.isalpha() for c in sys.argv[1])):
```

Comment:
No need to use comprehension here. `.isalpha()` is a string method and doing `sys.argv[1].isalpha()` would have had the same result.  
However you are forgetting to check for space character, here's how you could have done it:
```suggestion
if len(sys.argv) == 2 and sys.argv[1].replace(" ", "").isalpha():
```